courses/machine_learning/deepdive2/feature_engineering/solutions/3_Basic_Feature_Engineering_in_Keras.ipynb

```diff
@@ -0,0 +1,2565 @@
+{
```

Comment:
<p>you can decide if you want this to be a TODO or not. </p><p>In my mind this lab is about feature engineering, so I might just leave the TODOs to relate to that. They would have learned how to build a Keras sequential model in previous labs. </p><p>I can appreciate the want to reinforce old topics as well. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/777/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>